ISSUES.md

```diff
@@ -0,0 +1,98 @@
+# Detailed issue-reporting guidelines
+
+When reporting an issue, you should should try to provide:
+
+1. Succinct prose description of the problem (in addition to the issue title).
+
+2. Exact steps for reproducing the problem.
+
+3. Description of what you expect to see happen (the expected behavior) when you follow those steps.
+
+4. Description of what actually happens instead (the actual behavior) when you follow those steps.
+
+5. Either:
+
+   * The URL for a [reduced test case](#how-you-can-write-a-reduced-test-case) that reproduces the problem (e.g., using a site such as https://codepen.io/pen/, https://jsbin.com, or https://jsfiddle.net), or else at least the URL for a website/page that causes the problem.
+
+   * The HTML source for a reduced test case that reproduces the problem.
+
+6. Backtrace and Ladybird log output.
+
+7. Screenshot or screen recording.
+
+8. Operating system (Linux/macOS/Window/Android) in which you’re building or running Ladybird.
+
+9. Browser chrome of the build (Qt/AppKit/AndroidUI).
+
+10. Config/build flags you’re building with.
+
+11. Whether you’re interested yourself in trying to write a patch/PR with the fix for the problem.
+
+## How you can write a reduced test case
+
+When raising an issue, the single-most important thing you can provide is a reduced test others can use to reproduce the problem.
+
+You can create a reduced test case for a problem by starting from the HTML/JS/CSS source for a particular website/page which causes the problem, and removing as much content as possible while still having a document that reproduces the problem.
+
+Here’s how you can do that:
+
+1. Create a local copy of the page by either:
+
+   * using View Source in Ladybird to select the page source, and manually paste that source into a `REDUCTION.html` (or whatever name) file
+   * using `curl` or some similar means to grab and dump the page source to a `REDUCTION.html` (or whatever name) file:
+
+     ```
+     curl --output REDUCTION.html https://ladybird.org/
+     ```
```

Comment:
I would encourage people to use something like the [SingleFile](https://addons.mozilla.org/en-US/firefox/addon/single-file/) extension for Firefox or Chrome to save a canned copy of the page as it currently exists (post-JS-execution)

This also allows you to use Firefox/Chrome devtools to quickly strip out irrelevant elements from the page before saving a copy.

Reply:
Thanks — made it so.